content/06.discussion-a.md

```diff
@@ -1,82 +1,69 @@
+The default public nature of GitHub usage can add additional pressure to students learning to use the platform (Gomes et al. In Prep).   
+Thus, hesitancy may come from a more general reluctance to share data and code publicly [@doi:10.1038/s41597-021-00981-0], or technical and logistical issues related to storage of large data files and lack of integration with other research platforms.  
 
-### Using GitHub is a good start, but lots of practices to make repo more user friendly
+While many quantitative EEB researchers may take advantage of GitHub for individual use, collaborative use may lag behind due to how researchers traditionally divide labor within projects.  
+GitHub remains a tool predominantly used by computer scientists and software developers, and EEB researchers may view GitHub as a platform that only needs to be used by individuals writing code.      
+Those assumptions may have obscured the utility of GitHub for tasks other than traditional data analysis and software development, or how GitHub can facilitate the integration of code with non-coding aspects of projects through the practice of repository design.  
 
 
-*Contributors to this section: Ali, Emma* 
+### Box 1
+<!--*Contributors to this section: Ali, Emma* -->
 
 | 10 Tips for getting started in GitHub |
 |---|
-| 1. **Check for an existing solution to your problem.** The Github Help [webpage](https://docs.github.com/en) contains extensive and detailed documents with helpful screenshots. It is a good starting point for handling an issue, and has troubleshooting tips for specific problems. Alternatively, consider Tweeting your issue. There is a large community of GitHub users around the world who have likely faced analogous problems and may be able to provide quick solutions. Third, try to follow blogs e.g.(https://github.blog/), Twitter accounts or YouTube channels that regularly post practical solutions about common Github issues. |
+| 1. **Check for an existing solution to your problem** The GitHub Help [webpage](https://docs.github.com/en) contains extensive and detailed documents with helpful screenshots. It is a good starting point for handling an issue, and has troubleshooting tips for specific problems. Alternatively, consider Tweeting your issue. There is a large community of GitHub users around the world who have likely faced analogous problems and may be able to provide quick solutions. Third, try to follow blogs e.g.(https://github.blog/), Twitter accounts or YouTube channels that regularly post practical solutions about common GitHub issues. |
 | 2- **Consider taking free courses** such as those from [Software Carpentries](https://swcarpentry.github.io/git-novice/) and sharing these courses with your lab members or colleagues.|
```

Comment:
this number needs a period for consistency instead of a dash (also in hypothes.is comments)

Reply:
Made the update, thanks!